src/Xamarin.Android.Build.Tasks/Tasks/ProcessNativeLibraries.cs

```diff
@@ -13,19 +13,25 @@ namespace Xamarin.Android.Tasks
 	/// </summary>
 	public class ProcessNativeLibraries : AndroidTask
 	{
+		const string MonoComponentPrefix = "libmono-component-";
+
 		public override string TaskPrefix => "PRNL";
 
-		static readonly HashSet<string> DebugNativeLibraries = new HashSet<string> {
+		static readonly HashSet<string> DebugNativeLibraries = new HashSet<string> (StringComparer.OrdinalIgnoreCase) {
```

Comment:
We can probably remove this if we use the `RemoveDuplicates ` task on the ItemGroup before we pass it in.

https://github.com/xamarin/monodroid/blob/main/tools/msbuild/Xamarin.Android.Common.Debugging.targets#L600

https://docs.microsoft.com/en-us/visualstudio/msbuild/removeduplicates-task?view=vs-2019

Reply:
This particular set is used to exclude debug native libraries from the Release builds, so I think we should leave this in

Reply:
Sorry I meant the `_MonoComponents` one.

Reply:
You can also do `@(_MonoComponents->Distinct())` when passing it in to a task. This should also preserve the ordering if that matters.